### ISSUE TITLE
euler_zk_addr 分布式训练这个设置起到的作用？

### ISSUE THREAD

**Issue by 1oscar** [alibaba/euler#129]
想请教一下，分布式训练的时候，euler_zk_addr 设置这个起到的作用？
对于hadoop集群，只设置hdfs地址，不设置euler_zk_addr 参数，分布式能跑成功吗？

**Comment by yangsiran** [comment]
不同的euler实例之间依赖ZooKeeper做服务发现，所以euler_zk_addr和euler_zk_path在分布式训练时是必须的。

**Comment by 1oscar** [comment]
请教euler讨论的微信群有吗？

**Comment by 1oscar** [comment]
tf_euler/python/models/base.py文件
def decoder(self, embeddings, labels): 此函数，最后一行
return predictions, loss
可改为
_, auc = tf.metrics.auc(labels, tf.nn.sigmoid(logits), num_thresholds=5000)
    return predictions, loss,auc
增加demo 训练的时候auc的输出，这个你们的下一个发版应该会加上吧
